integrations/argocd/main.py

```diff
@@ -18,12 +18,27 @@ async def on_resources_resync(kind: str) -> list[dict[Any, Any]]:
     argocd_client = init_client()
 
     try:
-        return await argocd_client.get_resources(resource_kind=ObjectKind(kind))
+        if kind == ObjectKind.HISTORY:
+            return []
+        else:
```

Comment:
I would prefer having some enum class to which you refer as the resources with special handling, as we will probably will have more in the future. 

Take a look at this - https://github.com/port-labs/ocean/blob/54402315187ee5eab1f8d9083d0ac850b965dac6/integrations/azure/azure_integration/utils.py#L16 which implements exactly that.